# The marker positions in the trackable's frame of reference:
#
marker_positions:
  - x: -1.5
    y: -1.5
    z: 0.00
  - x: 1.5
    y: -1.5
    z: 0.00
  - x: 1.5
    y: 1.50
    z: 0.00
  - x: -1.5
    y: 1.50
    z: 0.00
